src/matrix_view.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+from src.matrix_operation import MatrixOperation
+
+
+class MatrixView:
+    def __init__(self, controller):
+        self._controller = controller
+        self._selected_operation = None
+        self._matrix1 = None
+        self._matrix2 = None
+
+    def start(self):
+        self._selected_operation = MatrixView.select_operation()
+        self.create_matrix(self._selected_operation)
+        self.perform_operation(self._matrix1, self._matrix2, self._selected_operation)
+
+    @staticmethod
+    def select_operation():
+        while True:
+            user_input = input(
+                "Enter the number corresponding to a matrix operation:\n "
+                "1 - Addition, "
+                "2 - Multiplication, "
+                "3 - Inverse:\n "
+            )
```

Comment:
nice opportunity to extract as a set of reusable menu handling classes, especially if you intended to increase the number of options and capabilities over time

Reply:
sure I'll consider this.